src/uniprotkb/components/entry/tabs/genomic-coordinates/Entries.tsx

```diff
@@ -0,0 +1,346 @@
+                  </Link>
+                </th>
+              ))}
+            </tr>
+          </thead>
+          <tbody>
+            {Object.entries(groupedExons).map(([id, exons]) => {
+              const location = (
+                <>
+                  {representativeEntry.gnCoordinate.genomicLocation
+                    .chromosome &&
+                    `${representativeEntry.gnCoordinate.genomicLocation.chromosome}:`}
+                  <LongNumber>
+                    {representativeEntry.gnCoordinate.genomicLocation
+                      .reverseStrand
+                      ? exons[0].genomeLocation.end.position
+                      : exons[0].genomeLocation.begin.position}
+                  </LongNumber>
+                  {' - '}
+                  <LongNumber>
+                    {representativeEntry.gnCoordinate.genomicLocation
+                      .reverseStrand
+                      ? exons[0].genomeLocation.begin.position
+                      : exons[0].genomeLocation.end.position}
+                  </LongNumber>
+                </>
+              );
+              return (
+                <tr key={id}>
+                  <td>
+                    {xrefInfo?.uriLink ? (
+                      <ExternalLink
+                        url={processUrlTemplate(xrefInfo.uriLink, { id })}
+                      >
+                        {id}
+                      </ExternalLink>
+                    ) : (
+                      id
+                    )}
+                  </td>
+                  <td>
+                    {xrefInfo ? (
+                      <ExternalLink
+                        url={getEnsemblLink(
+                          representativeEntry.taxid,
+                          representativeEntry.gnCoordinate.genomicLocation
+                            .reverseStrand
+                            ? exons[0].genomeLocation.end.position
+                            : exons[0].genomeLocation.begin.position,
+                          representativeEntry.gnCoordinate.genomicLocation
+                            .reverseStrand
+                            ? exons[0].genomeLocation.begin.position
+                            : exons[0].genomeLocation.end.position,
+                          representativeEntry.gnCoordinate.genomicLocation
+                            .chromosome
+                        )}
+                      >
+                        {location}
+                      </ExternalLink>
+                    ) : (
+                      location
+                    )}
+                  </td>
+                  {mappedIsoforms.map((isoformID) => {
+                    const exon = exons.find(
+                      (exon) => exon.accession === isoformID
+                    );
+                    if (!exon) {
+                      return <td key={isoformID}>-</td>;
+                    }
+                    return (
+                      <td key={isoformID}>
```

Comment:
I'm wondering if there is any way to make it more explicit to the user that this column is the protein coordinates? We have a header for the genomic coordinates but not for the proteins perhaps because it's a bit awkward as there can be multiple. I thought of a few ideas:

1. Use `title="protein coordinates for P05067-1 which map to 12:2323-24343"`
2. Have a header which spans all of the isoforms but left align the text to avoid the situation where a middle align disappears for wide tables/narrow screens.

Reply:
In the end, after the discussion in the UX thread, I've implemented a variant of you suggestion 1, on the column header.
We can always interate on that later on I guess